plugins/protocolbuffers/pyi/v31.0/buf.plugin.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1
+name: buf.build/protocolbuffers/pyi
+plugin_version: v31.0
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Python Stubs. Generates stub files for message and enum types.
+output_languages:
+  - python
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v31.0/LICENSE
+registry:
+  python:
+    package_type: "stub-only"
+    # https://github.com/protocolbuffers/protobuf/blob/v31.0/python/dist/setup.py#L76
+    requires_python: ">=3.9"
+    deps:
+      # https://pypi.org/project/protobuf/
+      - "protobuf~=6.31.0"
+      - "types-protobuf~=5.29"
```

Comment:
```suggestion
      - "types-protobuf~=6.30"
```

Reply:
Hope this will be okay.

Reply:
They usually lag behind - this is the latest version available at this time.